src/components/inputs/ComponentAttributeInput.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+  <q-input :model-value="localAttribute.value" :label="$t(localAttribute.name)"></q-input>
+</template>
+
+<script setup>
+import { defineProps, ref } from 'vue';
+
+const props = defineProps(['originalObject', 'attribute']);
+
+let localAttribute = props.originalObject.attributes.find((a) => a.name === props.attribute.name);
+if (!localAttribute) {
+  localAttribute = {
+    name: props.attribute.name,
+    type: props.attribute.type,
+    value: null,
```

Comment:
why you don't use the value of the attribute ?

Reply:
updated, but attribute value is expected to be undefined or null in this flow